standard/variables.md

```diff
@@ -265,9 +265,9 @@ The following categories of variables are classified as initially unassigned:
 
 #### 9.4.4.1 General
 
-In order to determine that each used variable is definitely assigned, the compiler shall use a process that is equivalent to the one described in this subclause.
+In order to determine that each used variable is definitely assigned, a compiler shall use a process that is equivalent to the one described in this subclause.
 
-The compiler processes the body of each function member that has one or more initially unassigned variables. For each initially unassigned variable *v*, the compiler determines a ***definite-assignment state*** for *v* at each of the following points in the function member:
+A compiler processes the body of each function member that has one or more initially unassigned variables. For each initially unassigned variable *v*, a compiler determines a ***definite-assignment state*** for *v* at each of the following points in the function member:
```

Comment:
These sentences possibly need a bit of work - or *maybe* just use "shall" a lot :)

Reply:
Reworded those sentences